b2/account_info.py

```diff
@@ -267,6 +302,9 @@ def _create_tables(self, conn):
            );
         """
         )
+        # This table is not used any more.  We may use it again
+        # someday if we same upload URLs across invocations of
```

Comment:
I meant "save":  "if we save upload URLs across invocations of"

I'll fix it.  Thanks.